pallets/pool-registry/src/mock.rs

```diff
@@ -176,7 +184,35 @@ impl pallet_pool_system::Config for Test {
 	type WeightInfo = ();
 }
 
-pub type Balance = u128;
+impl pallet_mock_change_guard::Config for Test {
+	type Change = pallet_pool_fees::types::Change<Test>;
+	type ChangeId = H256;
+	type PoolId = PoolId;
+}
+
+parameter_types! {
+	pub const MaxPoolFeesPerBucket: u32 = cfg_primitives::constants::MAX_POOL_FEES_PER_BUCKET;
+	pub const PoolFeesPalletId: PalletId = cfg_types::ids::POOL_FEES_PALLET_ID;
+}
+
+impl pallet_pool_fees::Config for Test {
```

Comment:
Yes, definitely. Not sure if its worth the overhead though because IMO `pallet-pool-system` should not use the mock to be closer to a real integration test.

Reply:
From a unitary testing point of view, I see it as a mock because we want to test here only `pool-registry` behavior.

If we want to be closer to real integration, maybe this test can be moved to `integration-tests` to assert that we are using a correct runtime configuration. It should be easy to move using the `RuntimeEnv`, just copying the whole test inside of `env.parachain_state_mut(|| {})` closure. There are currently some utilities to create pools and close epochs unter `generic/utils.rs`

But just a suggestion! I do not want to push more work on you if you feel oversaturated 🙏🏻